docs/how-to/input-output/gcs_duckdb.qmd

```diff
@@ -0,0 +1,24 @@
+---
+title: Loading Google Cloud Storage files with DuckDB
+---
+
+Loading data stored on Google Cloud Storage to DuckDB requires specifying `gcs` as filesystem.
+
+```
+from ibis.interactive import *
+
+con = ibis.duckdb.connect()
+
+import fsspec
+import gcsfs
+
+gcs = fsspec.filesystem("gcs")
+
+con.register_filesystem(gcs)
+
+t = con.read_json(<URL>)
+```
+
+Where `<URL>` is the url provided by GCS. Note that:
+- For private files use the URL with prefix `gs://` 
+- For public files you can use the URL with prefix `gcs://` 
```

Comment:
```suggestion
Where `<URL>` is the url provided by GCS.
```

Both `gs://` and `gcs://` should work regardless of permissions. Is that not the case for you?

Reply:
I decided to leave out the note altogther. I didn't want to include the specific URL. And figured it unncessary to specifiy the prefix. 

As for the access, I checked and yes, looks like all 3 (gs/gcs/storage.googleapis.com all work. Odd, I ran into issues the other day. Maybe mixed up the files: I had both a public and a private version of the file.